### PR TITLE
[#1940] Increasing Cypress Timeout Configuration

### LINKED ISSUE #1940: Increasing Cypress Timeout Configuration
### What feature(s) would you like to see in RepoSense

Running `./gradlew cypress` timeout before the cypress GUI shows up.

### If possible, describe the solution

The current `pageLoadTimeout` is **180000**. Testing on my local machine, I am unable to start cypress as it terminates due to premature timeout. A longer `pageLoadTimeout` will be better in this case.

According to [cypress documentation](https://docs.cypress.io/guides/references/configuration#Timeouts):
```
Network requests are limited by the underlying operating system, 
and may still time out if this value is increased.
```

### Additional context

Running on MacOS Ventura with Apple Silicon.

### PR DESCRIPTION
**Pull request by germainelee02**
Fixes #1940 

```
Currently the timeout to load pages is 18000. Hence, cypress 
is unable to start on some local machines as network request 
are limited to the underlying operating system, 
and can still timeout if the value is increased.

Let's increase the cypress timeout configuration.
```

### CODE REVIEW

**Comment by ckcherry23** [comment]
Hi @zhoukerrr, can I check if a `pageLoadTimeout` of `200000` is sufficient to start Cypress on your local machine without a premature timeout?

**Comment by zhoukerrr** [comment]
> Hi @zhoukerrr, can I check if a `pageLoadTimeout` of `200000` is sufficient to start Cypress on your local machine without a premature timeout?

Doubling will be better.

File: frontend/cypress/cypress.config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = defineConfig({
   fixturesFolder: false,
   video: false,
   defaultCommandTimeout: 30000,
-  pageLoadTimeout: 180000,
+  pageLoadTimeout: 200000,
```
Comment (ckcherry23) [inline frontend/cypress/cypress.config.js:7]:
```suggestion
  pageLoadTimeout: 400000,
```

**Review by ckcherry23 — APPROVED** [review]

**Review by chan-j-d — APPROVED** [review]
LGTM! @germainelee02 could you update the commit message to follow the maximum char limit 72 per line.

**Pull request merged**